Core/ProfitTrailer/StrategyHelper.cs

```diff
@@ -143,11 +160,8 @@ public static string GetStrategyShortcut(string strategyName, bool onlyValidStra
         case "rebuy timeout":
           result = "TIMEOUT";
           break;
-        case "min/max change perc":
```

Comment:
Should this be removed? Doesn't this show up if you set a max or min percentage change?

Reply:
Yeah, thanks.  It didn't work there -- i think because of the slash with ToLower.

Reply:
I also forgot to remove the rebuy timout as well.  So sloppy...  ;)

Reply:
BTW, min/max is only applied if a coin exceeds the set values, and can't be bought.